website/server/models/group.js

```diff
@@ -623,11 +642,20 @@ schema.methods.finishQuest = async function finishQuest (quest) {
     }
   });
 
-  let q = this._id === TAVERN_ID ? {} : {_id: {$in: this.getParticipatingQuestMembers()}};
+  let participants = this.getParticipatingQuestMembers();
```

Comment:
Hmm. I think this will not work with tavern quests then. I'd have to look at the getParticipatingQuestMembers method again, because I'm not sure how it interacts with a tavern quest.